tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml

```diff
@@ -110,19 +126,13 @@ jobs:
           allowPartiallySucceededBuilds: false
           artifact: 'onnxruntime_gpu'
           targetPath: '$(Build.SourcesDirectory)/onnxruntime/python/tools/tensorrt/perf/'
-      - script: 'docker pull $(dockerfile)'
+      - script: 'docker pull $(nvidia_dockerfile)'
         displayName: 'Pull Nvidia Dockerfile'
 
     - ${{ if eq(parameters.BuildORT, true) }}:
-
-      - ${{ if eq(parameters.RunNvidiaContainer, true) }}: 
-        - script: '$(Build.SourcesDirectory)/onnxruntime/python/tools/tensorrt/perf/build/build_image.sh -o $(Build.SourcesDirectory)/dockerfiles/Dockerfile.tensorrt -b $(branch) -t $(trtContainerVersion) -i ort-$(branch) -c 75 '
-          displayName: 'Build latest ORT Image in Nvidia Container'
-          workingDirectory: '$(Build.SourcesDirectory)/onnxruntime/python/tools/tensorrt/perf/build'
-      - ${{ else }}: 
-        - script: '$(Build.SourcesDirectory)/onnxruntime/python/tools/tensorrt/perf/build/build_image.sh -o $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/$(dockerfile) -b $(branch) -t $(trtContainerVersion) -i ort-$(branch) -c 75 '
-          displayName: 'Build latest ORT Image'
-          workingDirectory: '$(Build.SourcesDirectory)/onnxruntime/python/tools/tensorrt/perf/build'
+      - script: '$(Build.SourcesDirectory)/onnxruntime/python/tools/tensorrt/perf/build/build_image.sh -o $(Build.SourcesDirectory)/$(dockerfile) -b $(branch) -t $(tensorrt_version) -i $(image) -c 75 '
```

Comment:
I'm leaving this comment here because github won't allow me to leave a comment farther below:

I believe that line 143 (below) should also use the updated `$(image)` variable instead of `ort-$(branch)` as an argument to the `run_mem_test_docker.sh` script.

Reply:
Yes, correct fixed.